### PR TITLE
Conventional log folders

### PR DESCRIPTION
**Pull request by ozgunozerk**
Very trivial PR. 

Still, I put some comments below to put light onto changes and their reasonings

### CODE REVIEW

File: src-tauri/src/main.rs [hunk 1]
```diff
@@ -90,8 +90,15 @@ fn get_this_binary() -> PathBuf {
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    let ctx = tauri::generate_context!();
+    let id = &ctx.config().tauri.bundle.identifier;
```
Comment (ozgunozerk) [inline src-tauri/src/main.rs:94]: we need the bundler name, and only way is to use `generate_context`.

File: src-tauri/src/main.rs [hunk 2]
```diff
@@ -145,7 +152,7 @@ async fn main() -> Result<()> {
                 frontend_info_logger
             ],
         )
-        .build(tauri::generate_context!())
+        .build(ctx)
         .expect("error while running tauri application");
```
Comment (ozgunozerk) [inline src-tauri/src/main.rs:156]: that's why, we replaced this one with the previously generated context :)
I tested this locally, both in DEV and PROD, it worked.

File: src-tauri/Cargo.toml [hunk 3]
```diff
@@ -34,6 +34,7 @@ subspace-runtime = { git = "https://github.com/subspace/subspace", rev = "72b2c4
 subspace-service = { git = "https://github.com/subspace/subspace", rev = "72b2c44971a2db1266995a8c0fe1223ef09be784" }
 subspace-solving = { git = "https://github.com/subspace/subspace", rev = "72b2c44971a2db1266995a8c0fe1223ef09be784" }
 tokio = { version = "1.11.0", features = ["macros", "rt-multi-thread"] }
+dirs-next = "2.0.0"
```
Comment (nazar-pc) [inline src-tauri/Cargo.toml:37]: Why `dirs-next` and not more popular `dirs` that we use in node and farmer?

**Review by nazar-pc — APPROVED** [review]

File: src-tauri/Cargo.toml (hunk 3 above)
Comment (ozgunozerk) [reply src-tauri/Cargo.toml:37]: Did not know about that, will update accordingly, thanks!

**Review by nazar-pc — APPROVED** [review]

**Comment by nazar-pc** [comment]
On Linux this stored logs in in `/home/nazar-pc/.local/share/com.subspace.desktop` and the rest of data in `/home/nazar-pc/.local/share/subspace-desktop`, why? Why can't we have `subspace-desktop` instead of `com.subspace.desktop`?

**Pull request merged**